tests/manage/test_create_storage_class_pvc.py

```diff
@@ -1,135 +1,139 @@
         """
-        self.cephfs_secret = templating.load_yaml_to_dict(
-            constants.CSI_CEPHFS_SECRET_YAML
-        )
-        del self.cephfs_secret['data']['userID']
-        del self.cephfs_secret['data']['userKey']
-        self.cephfs_secret['data']['adminKey'] = (
-            get_admin_key_from_ceph_tools()
-        )
-        self.cephfs_secret['data']['adminID'] = constants.ADMIN_BASE64
-        logging.info(self.cephfs_secret)
-        secret = OCS(**self.cephfs_secret)
-        secret.create()
-        self.cephfs_sc = templating.load_yaml_to_dict(
-            constants.CSI_CEPHFS_STORAGECLASS_YAML
+        global RBD_PVC_OBJ, RBD_POD_OBJ, rbd_pv
+        log.info('creating pvc for RBD ')
+        pvc_name = helpers.create_unique_resource_name(
+            'test-rbd', 'pvc'
         )
-        self.cephfs_sc['parameters']['monitors'] = self.mons
-        self.cephfs_sc['parameters']['pool'] = (
-            f"{self.fs_data['metadata']['name']}-data0"
+        RBD_PVC_OBJ = helpers.create_pvc(RBD_SC_OBJ.name, pvc_name)
+        ocp_pv_list = helpers.get_all_pvs()
+        pv_list = ocp_pv_list['items']
+        for pv in pv_list:
+            if pv['spec']['claimRef']['name'] == RBD_PVC_OBJ.name:
+                rbd_pv = pv['metadata']['name']
+        RBD_POD_OBJ = helpers.create_pod(
+            constants.CEPHBLOCKPOOL, RBD_PVC_OBJ.name
         )
-        storage_class = OCS(**self.cephfs_sc)
-        storage_class.create()
-        self.cephfs_pvc = templating.load_yaml_to_dict(
-            constants.CSI_CEPHFS_PVC_YAML
-        )
-        pvc = PVC(**self.cephfs_pvc)
-        pvc.create()
-        log.info(pvc.status)
-        assert 'Bound' in pvc.status
-        pvc.delete()
-        storage_class.delete()
-        secret.delete()
 
     @pytest.mark.polarion_id("OCS-346")
-    def test_basics_cephfs(self):
+    def test_basics_cephfs(self, test_fixture_cephfs):
         """
         Testing basics: secret creation,
-         storage class creation  and pvc with rbd
+         storage class creation, pvc and pod with cephfs
         """
-        self.rbd_secret = templating.load_yaml_to_dict(
-            constants.CSI_RBD_SECRET_YAML
+        global CEPHFS_PVC_OBJ, CEPHFS_POD_OBJ, cephfs_pv
+        log.info('creating pvc for CephFS ')
+        pvc_name = helpers.create_unique_resource_name(
+            'test-cephfs', 'pvc'
+        )
+        CEPHFS_PVC_OBJ = helpers.create_pvc(
+            CEPHFS_SC_OBJ.name, pvc_name=pvc_name
         )
-        del self.rbd_secret['data']['kubernetes']
-        self.rbd_secret['data']['admin'] = get_admin_key_from_ceph_tools()
-        logging.info(self.rbd_secret)
-        secret = OCS(**self.rbd_secret)
-        secret.create()
-        self.rbd_sc = templating.load_yaml_to_dict(
-            constants.CSI_RBD_STORAGECLASS_YAML
+        ocp_pv_list = helpers.get_all_pvs()
+        pv_list = ocp_pv_list['items']
+        for pv in pv_list:
```

Comment:
same question - what are you trying to do here?

Reply:
I'm getting pv name of pvc, I could see a method backed_pv() would do that, so modifying the code. Thanks!